### DISCUSSION TITLE
VScode功能有點強大

### DISCUSSION THREAD

**Discussion by jerry850313** [1101-GS4719-C/Python-Community-GS4719-C#90] [Ideas]
最近用VScode跑深度學習，
原本都是直接用py，
然後我發現VScode也可以跑jupyter的檔案!!
![螢幕擷取畫面 2021-10-15 131727](https://user-images.githubusercontent.com/56184165/137436084-d4b921ac-54cd-448e-b73b-6c1c297a46e4.jpg)

**Comment by chia-kai-chang** [comment]
可以的唷 可以執行ipynb檔😃

**Comment by JunTingLin** [comment]
我也超愛vscode的 可以看看這篇[🆚打造舒服的vscode~幾個實用的插件&個人化設定 #520](https://github.com/NCU-GS4719-Python/Python-Community/discussions/520)

**Comment by PoooYen** [comment]
很多現役的IC設計工程師也都是用VS code喔~~

**Comment by Yuhsin000** [comment]
功能好像多到有點複雜，新手使用覺得焦慮，希望之後會熟悉

**Comment by treeann** [comment]
謝謝同學和老師有用的建議! 回去馬上試試! 希望安裝不會有問題

**Comment by minniewei** [comment]
我還以為jupyter的檔案也是.py檔!!!我真是太見識淺薄了!!

**Comment by chia-kai-chang** [comment]
分享剛剛看到的youtube影片，介紹10個程式設計師推薦的vscode套件：https://youtu.be/g1vy03ZY5mM

初步使用GitLens套件，功能相當強大，可以在圖形介面做很多git的設定，不需要打指令。另外，在編寫程式的時候，也可以看到程式碼區塊是由哪一位工程師提交的。
